processes/travel-policy.md

```diff
@@ -114,8 +114,10 @@ reservation!
 Every year, we publish a budget detailing the budget allowance and number of days you can use for
 conference conference. The conference list contains the number of days and cost for each event.
 
-For 2019, you can use up to **8 days** and **€3.400,00** for conferences. There are two cases where
-conference attendance doesn't count towards your budget:
+For 2019, you can use up to 8 working days and €3.400,00 for conferences. If you're in a part-time
+contract you've to reduce the monetary budget proportionally so, e.g. if you're in a 50% part-time
```

Comment:
"If you are working part-time, your budget is reduced proportionally"

Also, I would add a line break before "There are two cases where [...]"